### PR TITLE
adding firm business info

### PR DESCRIPTION
**Pull request by LowellBateman**
@johntowell @abuecker

### CODE REVIEW

File: api/v2/website/schema/website.res.json [hunk 1]
```diff
@@ -24,6 +24,38 @@
...
+          "type": "string"
+        },
+        "businessZip": {
+          "description": "Firm business zip",
+          "type": "string"
+        },
+        "businessCountry": {
+          "description": "Firm business country",
+          "type": "string"
+        },
```
Comment (johntowell) [inline api/v2/website/schema/website.res.json]: Are any of these required?

Comment (LowellBateman) [inline api/v2/website/schema/website.res.json]: yeah looks like theyre all required except address 2. *Updated

**Comment by johntowell** [comment]
Looks good, be careful when merging this can break platform. I would see if it breaks the rest tests first. And if it does have a PR to fix those up and then merged both at sameish time.

:+1:

**Comment by LowellBateman** [comment]
These aren't new fields, so it shouldn't be an issue.

platform pr here: https://github.com/trizic/platform/pull/394

**Pull request merged**